src/app/tests/integration/chip_im_responder.cpp

```diff
@@ -98,11 +100,44 @@ void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aC
 } // namespace chip
 
 namespace {
-
-// The CommandHandler object
 chip::TransportMgr<chip::Transport::UDP> gTransportManager;
 chip::SecureSessionMgr gSessionManager;
-chip::SecurePairingUsingTestSecret gTestPairing;
+chip::SecurePairingUsingTestSecret gTestPairing(chip::Optional<chip::NodeId>::Value(chip::kUndefinedNodeId),
+                                                static_cast<uint16_t>(0), static_cast<uint16_t>(0));
+LivenessEventGenerator gLivenessGenerator;
+
+uint64_t gDebugEventBuffer[256];
+uint64_t gInfoEventBuffer[256];
+uint64_t gCritEventBuffer[256];
+
+void InitializeEventLogging(chip::Messaging::ExchangeManager * apMgr)
+{
+    chip::app::reporting::LogStorageResources logStorageResources[] = {
+        { static_cast<void *>(&gDebugEventBuffer[0]), sizeof(gDebugEventBuffer), nullptr, 0, nullptr,
```

Comment:
is a static cast to void required? does just `gDebugEventBuffer` work?

What are the nullptr/0/nullptr values representing?

Reply:
yep, just removed LogStorageResources, nullptr/0/nullptr , they would be used in later persisted storage for eventing.
    chip::Platform::PersistedStorage::Key *
        mCounterKey;       
    uint32_t mCounterEpoch;.
    chip::PersistedCounter * mCounterStorage;